3-Write-your-own-network_en.ipynb

```diff
@@ -415,7 +408,8 @@
     "        h = F.relu(self.bn(self.conv(x)))\n",
     "        if self.pool_drop:\n",
     "            h = F.max_pooling_2d(h, 2, 2)\n",
-    "            h = F.dropout(h, ratio=0.25, train=self.train)\n",
+    "            chainer.using_config('train', self.train)\n",
```

Comment:
And [L:404](https://github.com/mitmul/chainer-handson/pull/4/files#diff-c18291b98c19eec59e574bc5dc3e3166R404) should be removed for v2. Because the train/test mode is managed by `chainer.config.train`, and it is automatically switched by `Trainer` and `Evaluator`.